### PR TITLE
Sl/sq fixes

### PR DESCRIPTION
**Pull request by SMLuthi**
### Fixes [BCDA-xxx](https://jira.cms.gov/browse/BCDA-xxx)

Sonarqube failed due to identifying a couple of "problem" spots.

### Proposed Changes

Make the changes Sonarqube proposes to avoid issues.

### Change Details

- Add table descriptions
- Replace `VALIGN` with css
- Add `scope` to table headers
- Update curl commands with a proper title (they arent token calls)
- Fix a header to be the proper size

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Acceptance Validation

Changes satisfy SQ.

### Feedback Requested

### CODE REVIEW

File: build.html [hunk 1]
```diff
@@ -145,9 +145,9 @@ <h1>
           </div>
         </div>
         <div id="request-metadata">
-          <h2>
+          <h1>
```
Comment (msnook) [inline build.html:148]: Will this pass the accessibility scan?

Comment (msnook) [reply build.html:148]: Might want to deploy your branch and then run this job: https://bcda-ci.adhocteam.us/job/BCDA%20-%20Static%20Site%20508%20Compliance/
Up to you; even if it does fail, we can fix it later.

Comment (SMLuthi) [reply build.html:148]: This should pass the scan as it doesnt seem to have any issues with the headers. But I will deploy to dev and run the quality gates to see the results.

**Review by jphechter — APPROVED** [review]
LGTM 👍

**Review by jphechter — APPROVED** [review]
👍 LGTM

**Pull request merged**